optimism2/chainlink/insert_price_feeds.sql

```diff
@@ -18,66 +19,76 @@ WITH gs AS (
     '1 hour'
     ) AS hr
     , "feed_name"
+    ,underlying_token_address
     FROM chainlink.oracle_addresses
 )
 
 , feed_updates AS (
-WITH feeds AS (
+SELECT
+dt, c.feed_name, c.price, c.proxy, c.address, underlying_token_address
+, c.price::decimal/(10^extra_decimals)::decimal AS underlying_token_price
+FROM (
 	SELECT
 	DATE_TRUNC('hour',block_time) AS dt
 	, feed_name
 	, AVG(bytea2numeric(topic2)::decimal/(10^decimals)::decimal) AS price
-	,"proxy", "address", underlying_token_address
+	,"proxy", "address"
 	FROM optimism.logs l
-	INNER JOIN chainlink.oracle_addresses cfa
+	INNER JOIN dune_user_generated.chainlink_feed_addresses cfa
```

Comment:
Trying to run this latest version. There is still have a reference to `dune_user_generated` here.

Reply:
Good catch, updated.